### PR TITLE
jdReplace: Add Recipe

### PR DESCRIPTION
**Pull request by JakobDev**
jdReplace is small mass replacement toll that I had written in 2019.

### CODE REVIEW

File: app-misc/jdreplace/jdreplace-2.2.recipe [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ARCHITECTURES="any"
+
+PROVIDES="
+	jdReplace = $portVersion
+	app:jdReplace = $portVersion
+	"
+
+REQUIRES="
+	haiku
+	python3
```
Comment (extrowerk) [inline app-misc/jdreplace/jdreplace-2.2.recipe:20]: `cmd:python3.8` would be the correct, i think.

**Comment by JakobDev** [comment]
@extrowerk Requirement is now fixed

File: app-misc/jdreplace/jdreplace-2.2.recipe [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://gitlab.com/JakobDev/jdReplace/-/archive/$portVersion/jdReplace-$portVersion.tar.gz"
+CHECKSUM_SHA256="8a8df34b3dd4c80ac3561f109162fb3ef81008a50ebc3ce3915ef0a3139e16b5"
+SOURCE_DIR="jdReplace-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	jdReplace = $portVersion
```
Comment (Begasus) [inline app-misc/jdreplace/jdreplace-2.2.recipe:14]: Should be (should reflect the recipe name):
```diff
- 	jdReplace = $portVersion
+ 	jdreplace = $portVersion
```

File: app-misc/jdreplace/jdreplace-2.2.recipe [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	app:jdReplace = $portVersion
+	"
+
+REQUIRES="
+	haiku
+	cmd:python38
```
Comment (Begasus) [inline app-misc/jdreplace/jdreplace-2.2.recipe:20]: Command is python3.8 (as @extrowerk mentioned earlier):
```diff
- 	cmd:python38
+ 	cmd:python3.8
```

**Comment by Begasus** [comment]
With the suggestions made it builds OK and starts fine (32-bit Haiku R1/Beta2)

**Comment by Begasus** [comment]
Thanks!

**Pull request merged**